<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MNotepad</class>
 <widget class="QMainWindow" name="MNotepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset resource="resfile.qrc">
    <normaloff>:/new/icons/icons/txt_file.png</normaloff>:/new/icons/icons/txt_file.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="wdgInsWord" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="tbHideWord">
         <property name="toolTip">
          <string>hide</string>
         </property>
         <property name="text">
          <string>hide</string>
         </property>
         <property name="icon">
          <iconset resource="resfile.qrc">
           <normaloff>:/new/icons/icons/stop.png</normaloff>:/new/icons/icons/stop.png</iconset>
         </property>
         <property name="shortcut">
          <string>Esc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CLatCopt" name="wdgInput" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btInsWord">
           <property name="text">
            <string>insert</string>
           </property>
           <property name="icon">
            <iconset resource="resfile.qrc">
             <normaloff>:/new/icons/icons/godown.png</normaloff>:/new/icons/icons/godown.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolButton" name="tbFindNext">
             <property name="toolTip">
              <string>find next</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resfile.qrc">
               <normaloff>:/new/icons/icons/downarrow.png</normaloff>:/new/icons/icons/downarrow.png</iconset>
             </property>
             <property name="shortcut">
              <string>Meta+PgDown</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonIconOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbFindPrev">
             <property name="toolTip">
              <string>find previous</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resfile.qrc">
               <normaloff>:/new/icons/icons/uparrow.png</normaloff>:/new/icons/icons/uparrow.png</iconset>
             </property>
             <property name="shortcut">
              <string>Meta+PgUp</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonIconOnly</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="txtText">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="plainText">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>264</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>&amp;action</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_all"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;view</string>
    </property>
    <addaction name="actionInsert_word"/>
    <addaction name="actionChange_font"/>
   </widget>
   <addaction name="menuAction"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>font</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/save.png</normaloff>:/new/icons/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/saveas.png</normaloff>:/new/icons/icons/saveas.png</iconset>
   </property>
   <property name="text">
    <string>s&amp;ave as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/cut.png</normaloff>:/new/icons/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/copy.png</normaloff>:/new/icons/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>c&amp;opy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/paste.png</normaloff>:/new/icons/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>&amp;select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/stop.png</normaloff>:/new/icons/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/txt_file.png</normaloff>:/new/icons/icons/txt_file.png</iconset>
   </property>
   <property name="text">
    <string>&amp;open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/refresh.png</normaloff>:/new/icons/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+R</string>
   </property>
  </action>
  <action name="actionInsert_word">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;insert/find word</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+I</string>
   </property>
  </action>
  <action name="actionChange_font">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;change font</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/leftarrow.png</normaloff>:/new/icons/icons/leftarrow.png</iconset>
   </property>
   <property name="text">
    <string>&amp;undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resfile.qrc">
     <normaloff>:/new/icons/icons/rightarrow.png</normaloff>:/new/icons/icons/rightarrow.png</iconset>
   </property>
   <property name="text">
    <string>&amp;redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>CLatCopt</class>
   <extends>QWidget</extends>
   <header>latcopt.h</header>
   <container>1</container>
   <slots>
    <signal>query()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections/>
</ui>
